R/facet-null.r

```diff
@@ -30,8 +30,10 @@ FacetNull <- ggproto("FacetNull", Facet,
     # Need the is.waive check for special case where no data, but aesthetics
     # are mapped to vectors
     if (is.waive(data) || empty(data))
-      return(cbind(data, PANEL = integer(0)))
-    data$PANEL <- 1L
+      return(cbind(data, PANEL = factor()))
```

Comment:
Looking at it afresh, this code doesn't seem right. Why are we `cbind`ing to something that's absent?